### PR TITLE
[RV64_DYNAREC] Added more opcodes

### PR DESCRIPTION
**Pull request by ksco**

### CODE REVIEW

File: src/dynarec/arm64/dynarec_arm64_00.c [hunk 1]
```diff
@@ -2422,7 +2422,7 @@ uintptr_t dynarec64_00(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int nin
                     INST_NAME("NEG Ed");
                     SETFLAGS(X_ALL, SF_SET_PENDING);
                     GETED(0);
-                    emit_neg32(dyn, ninst, rex, ed, x3, x4);
+                    emit_neg32(dyn, ninst, rex, ed, x2, x3, x4);
```
Comment (ksco) [inline src/dynarec/arm64/dynarec_arm64_00.c:2425]: Sorry for this, fixing it..

**Comment by ptitSeb** [comment]
I can review when you are done pushing, also. It's up to you.
PR can be larger now, the few steps are done, it's easier to debug now because new opcode are executed more often, as the dynarec blocks get larger and larger.

**Comment by ksco** [comment]
I'm stopping pushing with this PR, as I'm tracking down yet another bug reported by csmith, it might take some time.

**Comment by ptitSeb** [comment]
Ok, merging it then.

**Pull request merged**